components/content-search/index.js

```diff
@@ -343,12 +344,13 @@ const ContentSearch = ({
 	return (
 		<StyledComponentContext cacheKey="tenup-component-content-search">
 			<NavigableMenu onNavigate={handleOnNavigate} orientation="vertical">
-				<TextControl
-					label={label}
+				<SearchControl
```

Comment:
```suggestion
				<SearchControl
					__next40pxDefaultSize
```

In order to look closer to all the core component let also use the future default style now already :)